### PR TITLE
Feature/ui streams in admin

### PR DESCRIPTION
**Pull request by elfenheart**
add data under namespace sidebar in admin page. 

Able to create stream and edit stream properties from admin page.

### CODE REVIEW

File: cdap-ui/app/features/admin/routes.js [hunk 1]
```diff
@@ -143,6 +143,34 @@ angular.module(PKG.name + '.feature.admin')
               templateUrl: '/assets/features/admin/templates/namespace/datasets.html',
               controller: 'AdminDatasetsController'
             })
+              .state('admin.namespace.detail.datasets.streamcreate', {
```
Comment (ajainarayanan) [inline cdap-ui/app/features/admin/routes.js]: Shouldn't this be ```admin.namespace.detail.streams.create```?

I believe we either display both streams & datasets in admin view under 'data' or we have individual streams & datasets submenus under namespace to show them individually. Right?

File: cdap-ui/app/features/admin/controllers/DatasetsController.js [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+      _cdapPath: '/namespaces/' + $stateParams.nsadmin + '/data/datasets'
+    })
+      .then(function(res) {
+        $scope.datasets = res;
+      });
+
+    dataSrc.request({
+      _cdapPath: '/namespaces/' + $stateParams.nsadmin + '/streams'
+    }, function(res) {
+      if (angular.isArray(res) && res.length) {
```
Comment (ajainarayanan) [inline cdap-ui/app/features/admin/controllers/DatasetsController.js]: Any specific reason why we are having this check here alone and not in datasets? Do we need this check?

File: cdap-ui/app/features/admin/admin.less [hunk 3]
```diff
@@ -88,4 +88,10 @@ body.theme-cdap.state-admin {
       }
     }
   }
+
+  .modal-body {
```
Comment (ajainarayanan) [inline cdap-ui/app/features/admin/admin.less]: Any specific reason why we are customizing modal in admin view here?

Comment (elfenheart) [inline cdap-ui/app/features/admin/admin.less]: I am targeting the panel-title for the accordion (under modal). Without it, the text color is white. I am guessing this styling will get changed when we do the real styling for modals.

File: cdap-ui/app/features/admin/controllers/DatasetsController.js (hunk 2 above)
Comment (elfenheart) [inline cdap-ui/app/features/admin/controllers/DatasetsController.js]: updated

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**